### DISCUSSION TITLE
Building the Data System Error: GCAM, RStudio and One Drive

### DISCUSSION THREAD

**Discussion by ian-pimenta** [JGCRI/gcam-core#209] [Installing and running GCAM]
When installing Rstudio, if a user has OneDrive enabled, it seems like the installation path for R packages automatically sets to OneDrive/Documents. Having the "R" folder located within OneDrive can lead to several errors while trying to build GCAM input files. If you run into this issue, follow the guide below to relocate your "R" folder out of OneDrive.

**Comment by tjstelmach** [comment]
Hi @ian-pimenta, I was able to adjust the installation directory by modifying Rprofile file directly. Here's a useful resource: https://medium.com/@ValidScience/how-to-fix-rstudios-package-installation-on-windows-10-c1e602bf3a1f
